### ISSUE TITLE
About how to load parameters from other folder

### ISSUE THREAD

**Issue by polop08** [city96/ComfyUI_ExtraModels#30]
Thank you for answering my eariler questions. Now I have updated my pytorch and cuda version and use sd 1.5 models for vae. But due to my limited disk storage, I have to download Pixart model to the folder

`/data/Pixart_Alpha/PixArt-alpha`

Where the checkpoints, t5, vae are stored in the 

`/data/Pixart_Alpha/PixArt-alpha/t5/t5-v1_1-xxl`
`/data/Pixart_Alpha/PixArt-alpha/vae/sd-vae-ft-ema`

and I install comfy UI in 

`/home/polop`

I tried to modify extra_model_paths.yaml as

`base_path: /data/Pixart_Alpha/Pixart-alpha`
`checkpoints: /data/Pixart_Alpha/Pixart-alpha`
`vae: /data/Pixart_Alpha/Pixart-alpha/sd-vae-ft-ema`

However, that doesn't seem work for me. The file list in GUI is null and I can't find where to modify the path to t5 folder.
Could i ask how to modify the path to these data folders so that GUI could load parameter from them? Thank you for your help

**Comment by polop08** [comment]
Thank you for your previous help, the vae issue is solved.
But for this one, I am not sure if i did anything wrong, i modify the extra_model_path.yaml as

`PixArt:
    base_path: /data/Pixart_Alpha/Pixart-alpha
    t5: t5
    checkpoints: ./
    vae: vae
    configs: /home/ComfyUI/custom_nodes/ComfyUI_ExtraModels/PixArt`

When I download comfyUI, i just download it in my home folder since it is not a big file. Also, but when i download pixart model from the repo using git lfs, i download it in the folder 

`/data/Pixart_Alpha` 

and the file inside it will be like

`/data/Pixart_Alpha/Pixart-alpha/t5/t5-v1_1-xxl/pytorch_model-00001-of-00002.bin`
...
`/data/Pixart_Alpha/Pixart-alpha/vae/sd-vae-ft-ema/diffustion_pythorch_model.bin`
`/data/Pixart_Alpha/Pixart-alpha/PixArt-image-to-image-workflow.json`
...
`/data/Pixart_Alpha/Pixart-alpha/PixArt-XL-2-1024-MS.pth`
...

I put every file i get from 

`git clone https://huggingface.co/PixArt-alpha/PixArt-alpha`

And it turned out that the loader still can't find the correct path to the file

![image](https://github.com/city96/ComfyUI_ExtraModels/assets/168505779/68096739-3693-4978-a7dc-87c343679a45)

could i ask is there anything i could revise so that i could solve the issue? Thank you for your help!

**Comment by polop08** [comment]
Yes, I tried modify the yaml to the same as you wrote, and restart comfy 

`╰─$ python main.py
Total VRAM 48677 MB, total RAM 257776 MB
Set vram state to: NORMAL_VRAM
Device: cuda:0 NVIDIA RTX A6000 : cudaMallocAsync
VAE dtype: torch.bfloat16
Using pytorch cross attention
Adding extra search path t5 /data/Pixart_Alpha/Pixart-alpha/t5
Adding extra search path checkpoints /data/Pixart_Alpha/Pixart-alpha/./
Adding extra search path vae /data/Pixart_Alpha/Pixart-alpha/vae

Import times for custom nodes:
   0.0 seconds: /home/polop/ComfyUI/custom_nodes/websocket_image_save.py
   0.0 seconds: /home/polop/ComfyUI/custom_nodes/ComfyUI_ExtraModels

Starting server

To see the GUI go to: http://127.0.0.1:8188
`

Still, the loader cound't locate the .bin or .pth file, leaving only 'undifined' in my UI

![image](https://github.com/city96/ComfyUI_ExtraModels/assets/168505779/502d50bb-bac3-4b75-8ac6-d502d6720fe6)

how to define the path to the file for the loaders? thank you for all your assitance!

**Comment by city96** [comment]
Click where it says "unidentified" and select it from the list?

**Comment by polop08** [comment]
I click on it and still it shows undefined. it won't pop up a list of files to choose from but only show 'undefined' and nothing happens when i click on it.

**Comment by city96** [comment]
Huh, odd. The path is correct in the log from the info you provided so that's not it, and if you refreshed the UI then that shouldn't be the issue either.

Maybe you could try to set up a simple stable diffusion 1.5 workflow in base ComfyUI (without any custom nodes, and using the default workflow). I recommend you make separate folders for checkpoints/etc instead of reusing the PixArt folder to make it more clear, e.g. `/data/models/` as the base folder with subfolders like `checkpoints`, `vae`, etc.

If that also doesn't work, then I recommend opening an issue on the original ComfyUI repository.
If it *does* work then you can reenable the custom nodes. Move the PixArt checkpoint(s) to your new `/data/models/checkpoints` folders, and move T5 to `/data/models/t5` then add it to the extra paths yaml. If that is the step that breaks it, then it's an issue with my node and I'll investigate further.
